### ISSUE TITLE
Missing "SciSharp.TensorFlow.Redist-Windows-GPU" Version="2.4.1"

### ISSUE THREAD

**Issue by dje-dev** [SciSharp/TensorFlow.NET#751]
Attempts to run from latest committed code fail when switching to GPU version of Tensorflow because the package:
```
"SciSharp.TensorFlow.Redist-Windows-GPU" Version="2.4.1"
```
does not exist (yet!) and using the most recently available package:
```
"SciSharp.TensorFlow.Redist-Windows-GPU" Version="2.4.0"
```
fails with error:
```
Unable to find an entry point named 'TF_StringInit' in DLL 'tensorflow'.
```

Thanks!

**Comment by Oceania2018** [comment]
Havn't compile GPU version yet. Will do it this week.

**Comment by Oceania2018** [comment]
@dje-dev Currently I'm having issues when compiling Windows GPU version, basically it's related to CUDA driver confilict. I'll invest more time to solve it.

**Comment by Oceania2018** [comment]
Waiting for tensorflow v2.5.0 to release.

**Comment by dje-dev** [comment]
Good news, Tensorflow 2.5 is fully released now!

**Comment by Oceania2018** [comment]
@dje-dev Just released tensorflow v2.5.0 CPU/ GPU native library to [NuGet](https://www.nuget.org/packages/SciSharp.TensorFlow.Redist/2.5.0).
